Assets/MixedRealityToolkit.Services/InputAnimation/Editor/InputRecordingWindow.cs

```diff
@@ -204,12 +207,12 @@ private void DrawPlaybackGUI()
             {
                 if (GUILayout.Button("Load ..."))
```

Comment:
When I open the branch, it looks like the PlaybackService is null when I enter playmode and select the Playback tab which throws error in console 

This is in the HandInteractionRecordArticulatedHands example scene. This of course also blocks from me from trying to load a file